variables.tf

```diff
@@ -1,4 +1,454 @@
+         Possible values: `EQ`, `NE`, `LE`, `LT`, `GE`, or `GT`.
+      size:
+        The size, in bytes, to compare to the request part, after any transformations. 
+        Valid values are integers between `0` and `21474836480`, inclusive.
+      field_to_match:
+        The part of a web request that you want AWS WAF to inspect.
+        See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl#field-to-match
+      text_transformation:
+        Text transformations eliminate some of the unusual formatting that attackers use in web requests in an effort to bypass detection. 
+        See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl#text-transformation
+
+    visibility_config:
+      Defines and enables Amazon CloudWatch metrics and web request sample collection.
+      
+      cloudwatch_metrics_enabled:
+        Whether the associated resource sends metrics to CloudWatch.
+      metric_name:
+        A friendly name of the CloudWatch metric.
+      sampled_requests_enabled:
+        Whether AWS WAF should store a sampling of the web requests that match the rules.
+  DOC
+}
+
+variable "sqli_match_statement_rules" {
+  type        = list(any)
+  default     = null
+  description = <<-DOC
+    An SQL injection match condition identifies the part of web requests, 
+    such as the URI or the query string, that you want AWS WAF to inspect.
+    
+    action:
+      The action that AWS WAF should take on a web request when it matches the rule's statement.
+    name:
+      A friendly name of the rule.
+    priority:
+      If you define more than one Rule in a WebACL, 
+      AWS WAF evaluates each request against the rules in order based on the value of priority. 
+      AWS WAF processes rules with lower priority first.
+
+    statement: 
+      field_to_match:
+        The part of a web request that you want AWS WAF to inspect.
+        See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl#field-to-match
+      text_transformation:
+        Text transformations eliminate some of the unusual formatting that attackers use in web requests in an effort to bypass detection. 
+        See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl#text-transformation
+
+    visibility_config:
+      Defines and enables Amazon CloudWatch metrics and web request sample collection.
+      
+      cloudwatch_metrics_enabled:
+        Whether the associated resource sends metrics to CloudWatch.
+      metric_name:
+        A friendly name of the CloudWatch metric.
+      sampled_requests_enabled:
+        Whether AWS WAF should store a sampling of the web requests that match the rules.
+  DOC
+}
+
+variable "xss_match_statement_rules" {
+  type        = list(any)
+  default     = null
+  description = <<-DOC
+    An SQL injection match condition identifies the part of web requests, 
```

Comment:
```suggestion
    A cross-site scripting match condition identifies the part of web requests, 
```

Reply:
Please review the description of all variables.
E.g. 

xss_match_statement - (Optional) A rule statement that defines a cross-site scripting (XSS) match search for AWS WAF to apply to web requests. See XSS Match Statement below for details.